src/main/resources/org/jenkinsci/plugins/workflow/support/steps/stash/StashStep/help-allowEmpty.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    If false (default), an error will be raised when the stash does not contain files.
```

Comment:
Thanks for including the default!  I think this might make more sense configured a bit differently like:
```suggestion
    Create stash even if no files are included.  If false (default), an error will be raised when the stash does not contain files.
```